tracer/src/Datadog.Trace/AppSec/AsmRemoteConfigurationProducts.cs

```diff
@@ -0,0 +1,13 @@
+// <copyright file="AsmRemoteConfigurationProducts.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.AppSec;
+
+internal class AsmRemoteConfigurationProducts
+{
+    public static AsmFeaturesProduct AsmFeaturesProduct { get; } = new();
```

Comment:
Just for my understanding, are there going to be extra ASM "products" in here too _other_ than "features"? I guess I don't understand what the difference between a "feature" and a "product" is, if ASM is not a product?

Reply:
Remote config groups it's payloads into "products". The client will subscribe to a product and changes to this product will be pushed to the subscriber.

"FEATURES" was the name of a product. (Product names are always block capitals I guess the RC team hangs out on #ALL-CAPS-TYPING) The features product was design to contain the activation flags for different the different library subsystems (ASM, live debugger etc.), but we decided sharing config between different sub-systems was a bad idea for various reasons. This meant the features became exclusively owned by ASM, so we renamed it.